modules/bitgo/test/v2/unit/tss/ecdsa.ts

```diff
@@ -356,14 +365,13 @@ describe('Ecdsa tss helper functions tests', function () {
             const mockSendReq = { from: 'user', to: 'bitgo', share: config[index].mockShareToSendString } as SignatureShareRecord;
             const invalidSignatureShare = { from: 'bitgo', to: 'user', share: JSON.stringify(config[index].incorrectReceivedShareString) } as SignatureShareRecord;
             const nock = await nockSendSignatureShare({ walletId: wallet.id(), txRequestId: txRequest.txRequestId, signatureShare: mockSendReq, response: invalidSignatureShare }, 200);
-            await ECDSAMethods.sendShareToBitgo(bitgo, wallet.id(), txRequest.txRequestId, config[index].sendType, config[index].mockShareToSend, config[index].optionalDShare).should.be.rejectedWith(`Invalid ${config[index].shareReceived} from Bitgo`);
+            await ECDSAMethods.sendShareToBitgo(bitgo, wallet.id(), txRequest.txRequestId, config[index].sendType, config[index].mockShareToSend, config[index].optionalDShare).should.be.rejectedWith(/Excepted share length to be greater than or equal .*/g);
```

Comment:
We are not getting >= which length here in the error?

Reply:
since the length is different for each share, i'm using regex to get the general error part.